daq/host.py

```diff
@@ -290,7 +291,7 @@ def initialize(self):
         os.makedirs(self.scan_base)
         self._initialize_config()
         network = self.runner.network
-        if self.target_port:
+        if self.target_port and self._target_port_mirror:
```

Comment:
this should just be "self._use_target_port_mirror" and have any conditional logic where the variable is set.  Then you don't need to duplicate the logic in two places.

Reply:
Fixed